src/ceph-disk

```diff
@@ -1166,6 +1698,48 @@ def extract_parted_partition_numbers(partitions):
     numbers_as_strings = re.findall('^\d+', partitions, re.MULTILINE)
     return map(int, numbers_as_strings)
 
+
+# wait for udev event queue to clear
+def udev_settle():
+    time.sleep(1)
```

Comment:
Thanks for explaining the original rationale. udev must start as soon as the partition is created above. If there is a window of opportunity for udev events to be delayed, we need to wait for something that proves the desired events happened.